proc_square_refl.py

```diff
@@ -105,7 +105,7 @@ def filter_range(x): return array([j for j in x if
     
     #{{{ zeroth order phase correction
     for j in range(2):
-        ph0 = zeroth_order_ph(d['ch',j], fl=fl)
+        ph0 = zeroth_order_ph(d['ch',j], fl=None)
```

Comment:
for the channel 1 covariance test, it looks very odd. but maybe it's supposed to look like that? either way I just wanted to check with you because I believe this was the processing script we said had weird zeroth order phasing?

Reply:
so I'm pretty happy with the results now -- the covariance plots look so different b/c ch2 has the two blips w/ opposite phase while ch1 has everything w/ the same phase.  Ch2 has most points near zero, while ch1 has a lot of points far from 0.